src/reference/00-Getting-Started/03-Directories.md

```diff
@@ -61,7 +61,7 @@ form the complete build definition. See [organizing the build][Organizing-Build]
 ```
 build.sbt
 project/
-  Build.scala
+  Dependencies.scala
```

Comment:
Above it says

> `project` can contain `.scala` files, which are combined with `.sbt` files

which I find absolutely baffling since `.sbt` files are not Scala files and `.scala` source files don't compose. Do we really need to mention this in the Getting Started guide?